spec/functions_spec.sh

```
@@ -1,96 +1,96 @@
-                The status should eq 0
-            End
-            It 'outputs 0 or 1 depending a command exists or not'
-                When run check_cmd abcdefg
-                The status should eq 1
-                The stderr should eq "Please install abcdefg"
-            End
-        End
-    End
+Describe 'Testing src_path().'
+myrealpath() {
+  local LINK REALPATH
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)